src/core/cubic.c

```diff
@@ -404,13 +446,86 @@ CubicCongestionControlOnDataAcknowledged(
         goto Exit;
     }
 
+    //
+    // Update HyStart++ RTT sample.
+    //
+    if (AckEvent->MinRttValid) {
+        uint32_t Eta;
+
+        if (Cubic->HyStartState != HYSTART_DONE) {
+            //
+            // Update Min RTT for the first N ACKs.
+            //
+            if (Cubic->HyStartAckCount < QUIC_HYSTART_DEFAULT_N_SAMPLING) {
+                Cubic->MinRttInCurrentRound =
+                    CXPLAT_MIN(
+                        Cubic->MinRttInCurrentRound,
+                        AckEvent->MinRtt);
+                Cubic->HyStartAckCount++;
+            } else {
+                Eta =
```

Comment:
This is only used inside the next if statement. And all the branches necessary to evaluate the min/max should be easily combinable with the if on line 477.